protofire/omen-exchange
# feature/1861

closes https://github.com/protofire/omen-exchange/issues/1861

Keep in mind that the currently deployed airdrops on xdai and rinkeby are obviously test airdrops that award random test tokens, we will need to redeploy the airdrop contracts when the OMN token has been deployed.

If the address you are using to test did not receive an airdrop, you can grab an `airdropAddress` from the networks.ts file, e.g. on xdai the `airdropAddress` is `0x00aC027Def8227b60866b41e48Da30d23b656138`,  then find the associated airdrop .json file `services/airdrop/0x00aC027Def8227b60866b41e48Da30d23b656138.json` which will list all the addresses that are receiving an airdrop which you can use to test claiming functionality.

As mentioned on keybase we will integrate some of the airdrop ui into the guild modal in a later PR.

Note: If `airdrop` is set in local storage it will hide the airdrop modal, make sure to clear the `airdrop` local storage when testing 🙌

Steps to verify the airdrop json files:

1. Clone https://github.com/hexyls/omen-airdrop
2. `yarn install`
3. `yarn start`
4. The script can take a while to finish, maybe 5-10 mins
5. Verify that the generated mainnetProofs.json and xdaiProofs.json files match the airdrop json files in this PR
